src/docs/content/docs/configuration/reaper_specific.md

```diff
@@ -0,0 +1,343 @@
+the repair segment in question will be cancelled, if possible, and then scheduled for later
+repair again within the same repair run process.
+
+</br>
+
+### `incrementalRepair`
+
+Type: *Boolean*
+
+Default: *false*
+
+Sets the default repair type unless specifically defined for each run. Note that this is only supported with the PARALLEL repairParallelism setting. For more details in incremental repair, please refer to the following article.http://www.datastax.com/dev/blog/more-efficient-repairs
+
+*Note*: It is recommended to avoid using incremental repair before Cassandra 4.0 as subtle bugs can lead to overstreaming and cluster instabililty.
+
+</br>
+
+### `jmxAuth`
+
+Optional setting to allow Reaper to establish JMX connections to Cassandra clusters using password based JMX authentication. 
+
+    jmxAuth:
+      username: cassandra
+      password: cassandra
+
+#### `username`
+
+Type: *String*
+
+Cassandra JMX username.
+
+#### `password`
+
+Type: *String*
+
+Cassandra JMX password.
+
+</br>
+
+### `jmxConnectionTimeoutInSeconds`
+
+Type: *Integer*
+
+Default: *20*
+
+Controls the timeout for establishing JMX connections. The value should be low enough to avoid stalling simple operations in multi region clusters, but high enough to allow connections under normal conditions.
+
+</br>
+
+### `jmxPorts`
+
+Type: *Object*
+
+Optional mapping of custom JMX ports to use for individual hosts. The used default JMX port value is 7199. [CCM](https://github.com/pcmanus/ccm) users will find IP and port number to add in `~/.ccm/<cluster>/*/node.conf` or by running `ccm <node> show`.
+
+    jmxPorts:
+      127.0.0.1: 7100
+      127.0.0.2: 7200
+      127.0.0.3: 7300
+
+</br>
+
+### `localJmxMode`
+
+Type: *Boolean*
+
+Default: *false*
+
+Activates the mode where JMX is only accessible from localhost. If set to true, one Reaper instance must be running on each Cassandra node.
+
+</br>
+
+### `logging`
+
+Settings to configure Reaper logging.
+
+    logging:
+      level: INFO
+      loggers:
+        io.dropwizard: WARN
+        org.eclipse.jetty: WARN
+      appenders:
+        - type: console
+          logFormat: "%-6level [%d] [%t] %logger{5} - %msg %n"
+
+Definitions for some of the above sub-settings are as follows.
+
+#### `level`
+
+Type: *String*
+
+The log level to filter to. Where the level order is **ALL** < **DEBUG** < **INFO** < **WARN** < **ERROR** < **FATAL** < **OFF**. See the [log4j](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html) documentation for further information.
+
+#### `loggers`
+
+Type: *Object*
+
+Key value pair containing the logger class name as the key and other sub-settings as its value.
+
+#### `logFormat`
+
+Type: *String*
+
+The output format of an entry in the log.
+
+</br>
+
+### `metrics`
+
+Metrics configuration parameters for sending metrics to a reporting system via the [Dropwizard interface](http://www.dropwizard.io/1.1.4/docs/manual/configuration.html#metrics) .
```

Comment:
It would be good to specify that we ship the Graphite and DataDog reporters by default.

Reply:
Done. Add in a following sentence with that wording.